src/libraries/Common/src/System/Security/Cryptography/MLDsaCng.Windows.cs

```diff
@@ -246,51 +274,92 @@ protected override unsafe bool VerifyDataCore(ReadOnlySpan<byte> data, ReadOnlyS
             }
         }
 
-        internal static MLDsaCng ImportPkcs8PrivateKey(ReadOnlySpan<byte> source, out int bytesRead)
+        // TODO why can't we put this attribute on the class directly?
```

Comment:
The class isn't even supposed to be public.  Why are we putting annotations on it at all?